client/src/app/router.coffee

```diff
@@ -0,0 +1,24 @@
+Marionette = require('backbone.marionette')
+SiteView = require('./views/site')
+ListSitesView = require('./views/list-sites')
+FormView = require('./views/form')
+HomeView = require('./views/home')
+
+module.exports = Marionette.AppRouter.extend(
+
+  routes:
+    '': 'index'
+    'create': 'create'
+    'list_sites': 'list_sites'
+
+  index: ->
+    new HomeView().render()
```

Comment:
instead of rendering and having the el set for each root view, you should create a root region for your app

take a look at this pattern
https://github.com/samccone/edit.sx-frontend/blob/master/assets/js/_controller.coffee#L14